Updated 4.1. Node to Authorization Server Interactions.md (#38)
AMWA-TV/nmos-authorization-implementation-guide: docs/4.1. Node to Authorization Server Interactions.md

]
```

> Note that if you have multiple Authorization Servers there are some extra things to consider about [management of their corresponding public keys](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).

Reviewer: I think the wording isn't quite right here. 'You' is generally the implementer in this guide, but having many authorization servers is a deployment scenario. Either the spec says this is something all implementations are required /recommended to handle, or not?
```suggestion
Note that since there can be multiple Authorization Servers, there are some extra things to consider about [management of their corresponding public keys](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
```

Reviewer 2: LGTM
